api/config_entry_routes.go

```diff
@@ -38,45 +38,13 @@ type TCPRouteConfigEntry struct {
 	Namespace string `json:",omitempty"`
 }
 
-func (a *TCPRouteConfigEntry) GetKind() string {
-	return TCPRoute
-}
-
-func (a *TCPRouteConfigEntry) GetName() string {
-	if a != nil {
-		return ""
-	}
-	return a.Name
-}
-
-func (a *TCPRouteConfigEntry) GetPartition() string {
-	if a != nil {
-		return ""
-	}
-	return a.Partition
-}
-
-func (a *TCPRouteConfigEntry) GetNamespace() string {
-	if a != nil {
-		return ""
-	}
-	return a.GetNamespace()
-}
-
-func (a *TCPRouteConfigEntry) GetMeta() map[string]string {
-	if a != nil {
-		return nil
-	}
-	return a.GetMeta()
-}
-
-func (a *TCPRouteConfigEntry) GetCreateIndex() uint64 {
-	return a.CreateIndex
-}
-
-func (a *TCPRouteConfigEntry) GetModifyIndex() uint64 {
-	return a.ModifyIndex
-}
+func (a *TCPRouteConfigEntry) GetKind() string            { return TCPRoute }
+func (a *TCPRouteConfigEntry) GetName() string            { return a.Name }
+func (a *TCPRouteConfigEntry) GetPartition() string       { return a.Partition }
+func (a *TCPRouteConfigEntry) GetNamespace() string       { return a.GetNamespace() }
+func (a *TCPRouteConfigEntry) GetMeta() map[string]string { return a.GetMeta() }
```

Comment:
These are infinitely recursive

Reply:
Thanks for the catch, it should just be `a.Namespace`, etc. -- I'll fix it.